src/assets/scripts/client/scope/ScopeModel.js

```diff
@@ -126,6 +163,64 @@ export default class ScopeModel {
         return radarTargetModel.amendAltitude(altitude);
     }
 
+    /**
+     * Decrease the length of the PTL lines for all aircraft
+     *
+     * @for ScopeModel
+     * @method decreasePtlLength
+     */
+    decreasePtlLength() {
```

Comment:
you could reduce the code here to something like:
```js
const validValues = Object.keys(PTL_LENGTHS);
const currentIndex = validValues.indexOf(this._ptlLength);
const nextIndexValue = validValues[currentIndex + 1];

if (nextIndexValue < 0 || nextIndexValue > validValues.length) {
    nextIndexValue = 0;
}

return PTL_LENGTHS[nextIndexValue]
```
then you only really need a single method, call it `changePtlLength()`.  The caller(s) would need to be updated to pass a direction (increasing/decreasing), but that could be as simple as a (+1/-1).  You'd only have to change one line in my example:
```js
changePtlLength(direction) {
    // ...
    const nextIndexValue = validValues[currentIndex + direction];
    // ...
}
```

Reply:
Hm. Pretty clever, and more flexible. :+1:

Reply:
done